pkgs/development/python-modules/manimpango/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonPackage
+, pkg-config
+, fetchPypi
+, cairo
+, pango
+, pygobject3
+, glib
+, libintl
+, harfbuzz
+, cython
+, pythonOlder
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "manimpango";
+  version = "0.2.6";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    pname = "ManimPango";
+    inherit version;
+    sha256 = "64028b62b151bc80b047cc1757b27943498416dc4a85f073892a524b4d90ab41";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+  ];
+
+  buildInputs = [
+    harfbuzz
+    glib
+    cairo
+    libintl
+    pango
+  ];
+
+  propagatedBuildInputs = [
+    pygobject3
+  ];
+
+  setupPyBuildFlags = [
+    "-I${cairo.dev}/include/cairo"
+    "-L${cairo.out}/lib"
+    "-lcairo"
+    "-lpangocairo-1.0"
+  ];
```

Comment:
`setup.py` would get these flags from `pkg-config`, should you specify this manually?